llama_index/agent/openai_agent.py

```diff
@@ -402,7 +370,7 @@ def from_tools(
             callback_manager=callback_manager,
         )
 
-    def _get_tools(self, message: str) -> List[BaseTool]:
+    def _get_tools(self, message: str) -> list[BaseTool]:
```

Comment:
I think upper-case `List` is preferred for typing

Reply:
Cool, replaced with `(typing.)List`.